src/components/views/messages/TextualBody.js

```diff
@@ -77,24 +79,28 @@ module.exports = React.createClass({
         return (nextProps.mxEvent.getId() !== this.props.mxEvent.getId() ||
                 nextProps.highlights !== this.props.highlights ||
                 nextProps.highlightLink !== this.props.highlightLink ||
-                nextState.link !== this.state.link ||
+                nextState.links !== this.state.links ||
```

Comment:
By not doing a deep comparison, you're exploiting the fact that links is only ever set in `componentDidMount` (or rather, that it isn't modified once set). Could you add a comment to this effect?